core/platform/email/gae_email_services.py

```diff
@@ -34,6 +34,27 @@ def get_incoming_email_address(reply_to_id):
     return 'reply+%s@%s' % (reply_to_id, feconf.INCOMING_EMAILS_DOMAIN_NAME)
 
 
+def is_sender_email_valid(sender_email):
+    """Gets the sender_email address and validates it is of the form
+    'SENDER_NAME <SENDER_EMAIL_ADDRESS>'.
+
+    Args:
+        sender_email: str. The email address of the sender.
+
+    Returns:
+        False if sender_email is not valid.
```

Comment:
`bool. Whether the sender_email is valid.`

Reply:
Done Thanks!